fs/operations/operations.go

```diff
@@ -2091,6 +2091,21 @@ func SetTier(ctx context.Context, fsrc fs.Fs, tier string) error {
 	})
 }
 
+// SetTierFile changes tier of a single file in remote
+func SetTierFile(ctx context.Context, o fs.Object, tier string) error {
+	objImpl, ok := o.(fs.SetTierer)
```

Comment:
I normally call `objImpl` just `do` then to call it you say `do.SetTier()` which looks neat!

Reply:
Good call, updating